iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -119,6 +119,12 @@
 			<description>metadata extraction cache location, path pointing to
 				root cache directory holding meta.json file</description>
 		</property>
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
+            <value>$^</value>
+            <description>regexp matching funding class, describes projects which 
+            should be blacklisted by reference extraction algorithm</description>
+        </property>
```

Comment:
The problem is indentation looks good in eclipse when I perform changes but in fact in original file there are tabs while in newly introduced element - spaces. This misalignment causes troubles.